sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.projectVersion=0.0.1
 
 sonar.sources=src/app/
 sonar.test.inclusions=src/app/**/*.spec.ts, src/app/*.spec.ts
-sonar.exclusions= api/**, test/**, config/**, api/**/templates/**, src/public/**, src/environments/**, **/*.html, **/*.scss, src/app/config.service.server.spec.ts, src/app/config.service.server.ts, src/app/**/*.mock.ts, src/app/loader.service.ts
+sonar.exclusions= api/**, test/**, config/**, api/**/templates/**, src/public/**, src/environments/**, **/*.html, **/*.scss, src/app/config.service.server.spec.ts, src/app/config.service.server.ts, src/app/**/*.mock.ts, src/app/loader.service.ts, src/app/routing/pages/page-not-found/page-not-found.component.ts
```

Comment:
rather than an exhaustive list, remove these (loader.service.ts and page-not-found.component.ts) and just change the sonar.sources to be empty